node_modules/hubot-plusplus/src/plusplus.coffee

```diff
@@ -62,9 +62,10 @@ module.exports = (robot) ->
     # if we got a score, then display all the things and fire off events!
     if score?
       message = if reason?
-                  "#{name} has #{score} points, #{reasonScore} of which are for #{reason}."
+                  "#{name} has #{score} points, #{reasonScore} of which are for #{reason}"
```

Comment:
If you could pluralize that 'points', that'd be cool. :)

Not that coffeescript has an easy way to do that, though (that I can see).

http://autotelicum.github.io/Smooth-CoffeeScript/literate/pluralize.html